Dockerfile.ci

```diff
@@ -1,4 +1,4 @@
-FROM cvat/server
+FROM cvat/server:local
```

Comment:
This solution is fine for our CI workflows, but it can confuse developers who runs unit tests locally.
I would suggest such solution:
```docker
ARG CVAT_VERSION
FROM cvat/server:${CVAT_VERSION}
```
And add CVAT_VERSION argument into the docker-compose.ci.yml
```YAML
  cvat_ci:
    image: cvat_ci
    network_mode: host
    build:
      context: .
      dockerfile: Dockerfile.ci
      args:
        - CVAT_VERSION=${CVAT_VERSION:-latest}
```

But maybe it's not really important, @SpecLad What do you think?

Reply:
> This solution is fine for our CI workflows, but it can confuse developers who run unit tests locally.

But isn't this file specifically _for_ CI workflows? It's even called `Dockerfile.ci`. 🙂 

> I would suggest such solution:

Yeah, I thought of doing something like that. I didn't do it for a couple of reasons:

1. It didn't seem especially useful (as per above).
2. It makes regressions more likely, because if `CVAT_VERSION` it not added to the environment (for whatever reason), then it'll silently fall back to `:latest` again.

IMO, we should move in the other direction and make it so that developers use the `:local` tag by default. That would eliminate all confusion between published and development versions of the images. But that goes beyond fixing the immediate issue, so I didn't want to spend time on it.